stacker_blueprints/empire/daemon.py

```diff
@@ -275,18 +282,21 @@ def create_security_groups(self):
                 "ELBPort80FromTrustedNetwork",
                 IpProtocol="tcp", FromPort="80", ToPort="80",
                 CidrIp=Ref("TrustedNetwork"),
+                Condition="UseHTTP",
                 GroupId=Ref(ELB_SG_NAME)))
         t.add_resource(
             ec2.SecurityGroupIngress(
                 "ELBPort443FromTrustedNetwork",
                 IpProtocol="tcp", FromPort="443", ToPort="443",
                 CidrIp=Ref("TrustedNetwork"),
+                Condition="UseSSL",
                 GroupId=Ref(ELB_SG_NAME)))
         t.add_resource(
             ec2.SecurityGroupIngress(
                 "ELBPort443GitHub",
                 IpProtocol="tcp", FromPort="443", ToPort="443",
                 CidrIp=Ref("GitHubCIDR"),
+                Condition="UseSSL",
```

Comment:
We only allow Github hooks if we use SSL?  I'm guessing this is a limitation from github (and a good one)?